### DISCUSSION TITLE
Server-initiated mutations

### DISCUSSION THREAD

**Discussion by matthewgapp** [orbitinghail/sqlsync#41] [General]
I have an instance of the coordinator running on our server and I am using the pluggable reducer I wrote (https://github.com/orbitinghail/sqlsync/pull/40). But it's not quite enough, because I need to incorporate server-initiated mutations that are then synced to all clients. Is there an existing API I can use? Perhaps the `write_lsn` method?

**Comment by carlsverre** [comment]
This is an awesome feature request. I think I can put something together that solves this for you.
